django_opensearch_dsl/documents.py

```diff
@@ -196,18 +200,27 @@ def generate_id(cls, object_instance):
         """
         return object_instance.pk
 
-    def _prepare_action(self, object_instance, action):
-        return {
-            "_op_type": action,
+    def _prepare_action(self, object_instance, action, limit_fields):
```

Comment:
I'd rather you make the argument optional everywhere so that existing code calling/overriding these functions do not break.